release-schema.json

```diff
@@ -50,10 +50,38 @@
           "maximum": 1,
           "minimum": 0
         },
+        "competitiveMaximumPercentage": {
+          "title": "Maximum percentage of total value for competitive subcontracting",
+          "description": "Maximum percentage of the contract value the procuring entity asks the contractor to subcontract to third parties through a competitive procedure.",
+          "type": [
+            "number",
+            "null"
+          ],
+          "maximum": 1,
+          "minimum": 0.0001
+        },
+        "competitiveMinimumPercentage": {
+          "title": "Minimum percentage of total value for competitive subcontracting",
+          "description": "Minimum percentage of the contract value the procuring entity asks the contractor to subcontract to third parties through a competitive procedure.",
+          "type": [
+            "number",
+            "null"
+          ],
+          "maximum": 1,
+          "minimum": 0
+        },
         "value": {
           "title": "Subcontracted value",
           "description": "The estimated value of the part of the contract that the contractor will subcontract to third parties.",
           "$ref": "#/definitions/Value"
+        },
+        "competitive": {
+          "title": "Competitive subcontracting",
+          "description": "Whether the contractor will subcontract part of the contract to third parties through a competitive procedure.",
```

Comment:
Should we replace "contractor" with "supplier" throughout, to match OCDS terms?

Reply:
That makes sense, yes.